migrations/201706030000-add-HostId.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+/**
+ * Adds Transaction.HostId
+ * Adds User.isHost
+ * Adds User.currency
+ * 
+ * And populates them
+ */
+const Promise = require('bluebird');
+
+const cache = { HostIdForGroupId: {}, updatedHosts: {} };
+
+const updateHosts = (sequelize) => {
+
+  const getHostCurrency = (ug) => {
+    const query = `SELECT "txnCurrency" FROM "Transactions" WHERE "GroupId"=${ug.GroupId} AND "txnCurrency" is NOT NULL LIMIT 1`
```

Comment:
I'd limit this query to only consider donations that have a paymentMethod and aren't `deletedAt`. Expenses data for `txnCurrency` is unreliable.

Also, would be good to run this locally and spit out what it's setting for each host and make sure that anyone who isn't set to USD, we eyeball manually.